demo/demo-metadata.yml

```diff
@@ -0,0 +1,15 @@
+title:
+  Datasette Nteract Data Explorer Demo
+
+description_html:
```

Comment:
I really like this string by default with an optional `_html` extension API. If these blocks get longer, I'll store the description in markdown somewhere, and convert them to HTML in a build step.